vendor/github.com/getkin/kin-openapi/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
If we run the generate step as part of verifying changes in CI, but not as part of building the RPM, could we avoid vendoring the dependencies for the tool entirely? We might want to skip vendoring these dependencies even if we make the separate go module mentioned in an earlier comment.